<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RpnKeypadDialog</class>
 <widget class="QDialog" name="RpnKeypadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RPN Calc</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="2" column="9">
      <widget class="QPushButton" name="button_push_xy">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>XY→</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QPushButton" name="button_sum">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="button_multiplication">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <widget class="QPushButton" name="button_push_y">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>Y→</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="button_dot">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="9" column="5">
      <widget class="QPushButton" name="button_tan">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>TAN</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="button_1">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QPushButton" name="button_push_x">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>X→</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="button_sin">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>SIN</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="button_go_xy">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>XY Go</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="button_roll_up">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>ROL↑</string>
       </property>
      </widget>
     </item>
     <item row="10" column="9">
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QPushButton" name="button_go_z">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>Z Go</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="button_back">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>⌫</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="button_4">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="button_0">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QPushButton" name="button_sqrt">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>√</string>
       </property>
      </widget>
     </item>
     <item row="10" column="4" colspan="2">
      <widget class="QPushButton" name="button_enter">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>Enter⏎</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="button_roll_down">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>ROL↓</string>
       </property>
      </widget>
     </item>
     <item row="6" column="9">
      <widget class="QPushButton" name="button_push_spindle">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>Spindle→</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="button_set_x">
       <property name="styleSheet">
        <string>    font: bold;</string>
       </property>
       <property name="text">
        <string>→X</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="button_8">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="button_swap">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>Swap</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QPushButton" name="button_cos_2">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>COS</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="button_set_xy">
       <property name="styleSheet">
        <string>    font: bold;</string>
       </property>
       <property name="text">
        <string>→XY</string>
       </property>
      </widget>
     </item>
     <item row="5" column="9">
      <widget class="QPushButton" name="button_push_z">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>Z→</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="button_5">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QPushButton" name="button_cos">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>ATAN2</string>
       </property>
      </widget>
     </item>
     <item row="7" column="7">
      <widget class="QPushButton" name="button_set_feed">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>→Feed</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="9" column="3">
      <widget class="QPushButton" name="button_subtract">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QPushButton" name="button_set_feed_2">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>X²</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QPushButton" name="button_set_z">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>→Z</string>
       </property>
      </widget>
     </item>
     <item row="6" column="7">
      <widget class="QPushButton" name="button_set_spindle">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>→Spindle</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="button_2">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="button_9">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="button_set_y">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>→Y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="button_go_x">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>X Go</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="button_7">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="7" column="9">
      <widget class="QPushButton" name="button_push_feed">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>Feed→</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QPushButton" name="button_go_y">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string> Y Go</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="button_division">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="button_swap_3">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>DUP2</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QPushButton" name="button_plus_minus">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>±</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QPushButton" name="button_3">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="button_6">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="button_swap_2">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>Over</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="6" colspan="5">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="toggle_jog_cut">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>Jog</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="toggle_relative_absolute">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>Relative</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="toggle_work_machine">
       <property name="styleSheet">
        <string notr="true">    font: bold;</string>
       </property>
       <property name="text">
        <string>Work</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
